tests/test_redirects.py

```diff
@@ -0,0 +1,211 @@
+from test_grapple import BaseGrappleTest
+from testapp.factories import RedirectFactory
+from testapp.models import BlogPage
+from wagtail_factories import SiteFactory
+
+
+class TestRedirectQueries(BaseGrappleTest):
+    @classmethod
+    def setUpTestData(cls) -> None:
+        super().setUpTestData()
+        cls.page = BlogPage(title="Test page", slug="test-page-url", date="2020-01-01")
+        cls.home.add_child(instance=cls.page)
+
+    def test_basic_query(self):
+        """
+        Test that Redirect fields can be queried through graphql.
+        """
+
+        self.redirect = RedirectFactory(
```

Comment:
By using `self`, you are modifying data held in the class, which is not what you want here. You are creating something for use just within this method, which is achieved by not including `self`, e.g.:

```
redirect = RedirectFactory(...)
```

However, you are not reusing `redirect` anywhere, so there's no need to assign it to a variable. Instead, you can just do:

```
RedirectFactory(...)
```

Reply:
Addressed in 40eeb98e50bd5cf9895d758fbd761f3f2c474567